packages/tpetra/core/test/CrsGraph/CrsGraph_UnitTests_Swap.cpp

```diff
@@ -160,20 +162,21 @@ generate_crsgraph(Teuchos::RCP<comm_t>                      comm,
     for(auto& p: gbl_row_owners) { gbl_row2pid.insert(p); }
 
     // Print out some debugging information on what's in the graph
-    #if 0
-    std::cout << "p=0 | gbl_num_rows: " << gbl_rows.size() << std::endl;
-    for(auto& p: gbl_rows)
+    if(verbose)
     {
-        std::cout << "p=0 | gbl_row     : " << p.first << " (" << p.second.size() << ") ";
-        for(auto& j: p.second)
+        std::cout << "p=0 | gbl_num_rows: " << gbl_rows.size() << std::endl;
```

Comment:
For future reference (not for this PR), when printing debugging messages nondeterministically over multiple MPI processes to cout or cerr, good practice is to assemble each line (including the endline character) from each process as a string.  This tends to make messages from different processes less mixed up.  For example:
```c++
std::ostringstream os;
os << "(Proc " << myRank << ") gbl_num_rows: " << gbl_rows.size () << std::endl;
std::cout << os.str ();
```

Reply:
At some point, it would make sense to turn that into a macro:
```c++
#define TPETRA_PRINTLN_LOCAL( out, myRank, stuff ) \
  do { \
    std::ostringstream tpl_out; \
    tpl_out << "(Proc " << myRank << ") " << stuff << std::endl; \
    out << tpl_out.str (); \
  } while (false);
```
Alas, C++ lacks the equivalent of Common Lisp's `GENSYM`, so I can't guarantee that `tpl_out` doesn't shadow any other names in scope.  This should still work, though.

Reply:
Is the `#define TPETRA_PRINTLN_LOCAL` a thing that already exists, or are you suggesting creating this macro in the file to use?  Either way is doable, but if there's already a macro or function defined that does this then I'm happy to use it :)

Reply:
@william76 I'm just suggesting this as a later thing to think about, after this PR is done :-D  Thanks though!